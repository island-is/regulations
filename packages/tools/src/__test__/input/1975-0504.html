<div class="Section1">
  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">1. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Sveitarstjórn er heimilt að leggja gatnagerðargjöld á lóðir í sveitarfélaginu, eftir því sem nánar segir í reglugerð þessari. Heimilt er að leggja gatnagerðargjald á land, sem sambærilegt telst, þótt ekki sé sérstök 1óð.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">2. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Gatnagerðargjöld eru tvenns konar, A-gjald sem er gjald vegna þátttöku lóðarhafa í kostnaði við að gera undirbyggða götu og B-gjald, sem er gjald vegna þátttöku lóðarhafa í þvi að gera bundið slitlag á götu svo og að ganga endanlega frá gangstétt.</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">3. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Nú ákveður sveitarstjórn að leggja á A-gjald vegna nýbyggingar, og skal þá miða gjaldið við ákveðinn hluta byggingarkostnaðar a rúmmetra, eins og hann er talinn of Hagstofu Íslands svo sem hér segir:</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Einbýlishús<span style="&#39mso-tab-count:">                                                                                                                   </span> 2.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Tvíbýlishús og sambyggð einbýlishús á einni hæð<span style="&#39mso-tab-count:5&#39;">                                                           </span> 1.5%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Tveggja hæða raðhús og íbúðarhús<span style="&#39mso-tab-count:7&#39;">                                                                               </span> 1.5%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Fjölbýlishús <span style="&#39mso-tab-count:">                                                                                                                </span> 1.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Verslunar- og skrifstofubyggingar<span style="&#39mso-tab-count:7&#39;">                                                                                 </span> 2.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Önnur hús og byggingar<span style="&#39mso-tab-count:8&#39;">                                                                                               </span> 1.5%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Af kjöllurum (jarðhæðum) og rishæðum, sem. ekki eru notuð til ibúðar eða atvinnurekstrar svo og bifreiðageymslum skal greiða hálft gjald.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Auk þessa gjalds skal lóðarhafi greiða kr. 40.00 of hverjum fermetra lóðar. Gjald skal miða við stærð húss skv. samþykktum uppdrætti. Nú er ekki fyrir<span style="&#39mso-tab-count:1&#39;">           </span> hendi samþykktur uppdráttur, og skal þá miða við stærð húss sem skipulag gerir ráð fyrir, en sé slíkt ekki ákveðið í skipulagi skal miða við eftirtaldar lágmarks stærðir:</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Einbýlishús með tilheyrandi bílgeymslu<span style="&#39mso-tab-count:6&#39;">                                                              </span> 500m<sup>3</sup></span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Raðhús, tvíbýlishús, sambyggð einbýlishús, hver íbúð<span style="&#39mso-tab-count:4&#39;">                                        </span> 400m<sup>3</sup></span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Fjölbýlishús<span style="&#39mso-tab-count:">                                                                                                     </span> 300m<sup>3</sup></span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Í atvinnuhúsnæði skal við útreikning rúmmáls til gatnagerðargjalds miðað við lofthæð 3.3 m að innanmáli, nema uppdráttur sýni annað.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="EN-US">Framangreindar reglur gilda, ef hús er stækkað, að því er til stækkunar tekur.</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">4. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Nú ákveður sveitarstjórn að leggja á B-gjald til að ljúka frágangi götu, umfram það sem segir i 3. gr., þ. e. til að gera slitlag og gangstétt, og skal þá miða gjaldið við ákveðinn hluta byggingarkostnaðar á rúmmetra eins og hann er talinn of Hagstofu Íslands svo sem hér segir:</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Einbýlishús<span style="&#39mso-tab-count:">                                                                                                                   </span> 4.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Tvíbýlishús og sambyggð einbýlishús á einni hæð<span style="&#39mso-tab-count:5&#39;">                                                           </span> 3.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Tveggja hæða raðhús og íbúðarhús<span style="&#39mso-tab-count:7&#39;">                                                                               </span> 3.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Fjölbýlishús<span style="&#39mso-tab-count:">                                                                                                                 </span> 2.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Verslunar- og skrifstofubyggingar<span style="&#39mso-tab-count:7&#39;">                                                                                 </span> 4.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Önnur hús og byggingar<span style="&#39mso-tab-count:8&#39;">                                                                                               </span> 3.0%</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Auk þessa gjalds skal lóðarhafi greiða í gatnagerðargjöld kr. 60.00 of hverjum fermetra lóðar.</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">5. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Gjöld samkvæmt 3. gr. breytast árlega samkvæmt vísitölu byggingarkostnaðar, sem gildir um áramót.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">6. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="EN-US">Gjalddagar gatnagerðargjalds skv. 3. gr. skulu vera sem hér segir:</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Ef um er að ræða leigulóð í eigu sveitarfélagsins. getur sveitarstjórn áskilið, að úthlutun verði háð því, að full skil verði gerð innan ákveðins frests, áður en byggingarleyfi verður veitt eða sérstakur samningur gerður um greiðslur.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">7. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Gatnagerðargjald skv. 4. gr. miðast við þá vísitölu byggingarkostnaðar, sem í gildi er, þegar lagning bundins slitlags fer fram. Nú breytist vísitala, meðan verk fer fram, og skal þá miða við meðalvísitölu á því tímabili, sem um er að ræða.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">8. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Nú ákveður sveitarstjórn að innheimta gjald, skv. 4. gr., af fasteignum við götu, þar sem þegar hefur verið lagt slitlag og gangstétt, skal miða gjaldið við þá vísitölu, sem í gildi er, þegar samþykkt þessi öðlast gildi.</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">9. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Gatnagerðargjald samkvæmt 4. gr. greiðist þannig að 20% greiðist, þegar lagningu bundins slitlags er lokið. Eftirstöðvar greiðast með jöfnum árlegum afborgunum á næstu 4 árum.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="EN-US">Gjalddagi afborgana er 1. júli. Vextir 10% greiðast eftir á sömu gjalddögum.</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US"> </span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">10. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Sveitarstjórn er heimilt að hækka eða lækka gjaldstuðla samkv. 3. gr. og 4, gr. hvern um sig eða alla í einu um allt að 50% án frekari staðfestingar ráðuneytis.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">11. gr.</span></strong></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Gatnagerðargjaldi fylgir lögveð í fasteign þeirri sem það er lagt á, og gengur það ásamt vöxtum fyrir öllum veðkröfum, sem á eigninni hvíla og tekur einnig til vátryggingarfjár eignarinnar.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US">Heimilt er að taka gjöld samkvæmt lögum þessum lögtaki skv. lögum nr. 29/1885.</span></p>

  <p class="MsoBodyText"><span lang="EN-US">Samþykkt þessi er samin og samþykkt of hreppsnefnd Hvolshrepps og staðfestist hér með samkvæmt lögum um gatnagerðargjöld nr. 51/1974 sbr. lög nr. 31/1975 til að öðlast gildi þegar í stað.</span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> </span></p>

  <p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="EN-US"> Félagsmálaráðuneytið, 18. nóvember 1975</span></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><em><span lang="EN-US"> </span></em></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><em><span lang="EN-US"> </span></em></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">F. h. r.</span></strong></p>

  <p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="EN-US">Jón S. Ólafsson.</span></strong></p>
  </div>
