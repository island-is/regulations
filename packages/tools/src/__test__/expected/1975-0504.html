<h3 class="article__title">
1.
gr.
</h3>

<p>
Sveitarstjórn
er
heimilt
að
leggja
gatnagerðargjöld
á
lóðir
í
sveitarfélaginu,
eftir
því
sem
nánar
segir
í
reglugerð
þessari.
Heimilt
er
að
leggja
gatnagerðargjald
á
land,
sem
sambærilegt
telst,
þótt
ekki
sé
sérstök
1óð.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Gatnagerðargjöld
eru
tvenns
konar,
A-gjald
sem
er
gjald
vegna
þátttöku
lóðarhafa
í
kostnaði
við
að
gera
undirbyggða
götu
og
B-gjald,
sem
er
gjald
vegna
þátttöku
lóðarhafa
í
þvi
að
gera
bundið
slitlag
á
götu
svo
og
að
ganga
endanlega
frá
gangstétt.
</p>

<h3 class="article__title">
3.
gr.
</h3>

<p>
Nú
ákveður
sveitarstjórn
að
leggja
á
A-gjald
vegna
nýbyggingar,
og
skal
þá
miða
gjaldið
við
ákveðinn
hluta
byggingarkostnaðar
a
rúmmetra,
eins
og
hann
er
talinn
of
Hagstofu
Íslands
svo
sem
hér
segir:
</p>

<p>
Einbýlishús
<span data-legacy-indenter="">
                                                                                                                   
</span>
2.0%
</p>

<p>
Tvíbýlishús
og
sambyggð
einbýlishús
á
einni
hæð
<span data-legacy-indenter="">
                                                           
</span>
1.5%
</p>

<p>
Tveggja
hæða
raðhús
og
íbúðarhús
<span data-legacy-indenter="">
                                                                               
</span>
1.5%
</p>

<p>
Fjölbýlishús
<span data-legacy-indenter="">
                                                                                                                
</span>
1.0%
</p>

<p>
Verslunar-
og
skrifstofubyggingar
<span data-legacy-indenter="">
                                                                                 
</span>
2.0%
</p>

<p>
Önnur
hús
og
byggingar
<span data-legacy-indenter="">
                                                                                               
</span>
1.5%
</p>

<p>
Af
kjöllurum
(jarðhæðum)
og
rishæðum,
sem.
ekki
eru
notuð
til
ibúðar
eða
atvinnurekstrar
svo
og
bifreiðageymslum
skal
greiða
hálft
gjald.
</p>

<p>
Auk
þessa
gjalds
skal
lóðarhafi
greiða
kr.
40.00
of
hverjum
fermetra
lóðar.
Gjald
skal
miða
við
stærð
húss
skv.
samþykktum
uppdrætti.
Nú
er
ekki
fyrir
<span data-legacy-indenter="">
           
</span>
hendi
samþykktur
uppdráttur,
og
skal
þá
miða
við
stærð
húss
sem
skipulag
gerir
ráð
fyrir,
en
sé
slíkt
ekki
ákveðið
í
skipulagi
skal
miða
við
eftirtaldar
lágmarks
stærðir:
</p>

<p>
Einbýlishús
með
tilheyrandi
bílgeymslu
<span data-legacy-indenter="">
                                                              
</span>
500m<sup>3</sup>
</p>

<p>
Raðhús,
tvíbýlishús,
sambyggð
einbýlishús,
hver
íbúð
<span data-legacy-indenter="">
                                        
</span>
400m<sup>3</sup>
</p>

<p>
Fjölbýlishús
<span data-legacy-indenter="">
                                                                                                     
</span>
300m<sup>3</sup>
</p>

<p>
Í
atvinnuhúsnæði
skal
við
útreikning
rúmmáls
til
gatnagerðargjalds
miðað
við
lofthæð
3.3
m
að
innanmáli,
nema
uppdráttur
sýni
annað.
</p>

<p>
Framangreindar
reglur
gilda,
ef
hús
er
stækkað,
að
því
er
til
stækkunar
tekur.
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Nú
ákveður
sveitarstjórn
að
leggja
á
B-gjald
til
að
ljúka
frágangi
götu,
umfram
það
sem
segir
i
3.
gr.,
þ.
e.
til
að
gera
slitlag
og
gangstétt,
og
skal
þá
miða
gjaldið
við
ákveðinn
hluta
byggingarkostnaðar
á
rúmmetra
eins
og
hann
er
talinn
of
Hagstofu
Íslands
svo
sem
hér
segir:
</p>

<p>
Einbýlishús
<span data-legacy-indenter="">
                                                                                                                   
</span>
4.0%
</p>

<p>
Tvíbýlishús
og
sambyggð
einbýlishús
á
einni
hæð
<span data-legacy-indenter="">
                                                           
</span>
3.0%
</p>

<p>
Tveggja
hæða
raðhús
og
íbúðarhús
<span data-legacy-indenter="">
                                                                               
</span>
3.0%
</p>

<p>
Fjölbýlishús
<span data-legacy-indenter="">
                                                                                                                 
</span>
2.0%
</p>

<p>
Verslunar-
og
skrifstofubyggingar
<span data-legacy-indenter="">
                                                                                 
</span>
4.0%
</p>

<p>
Önnur
hús
og
byggingar
<span data-legacy-indenter="">
                                                                                               
</span>
3.0%
</p>

<p>
Auk
þessa
gjalds
skal
lóðarhafi
greiða
í
gatnagerðargjöld
kr.
60.00
of
hverjum
fermetra
lóðar.
</p>

<h3 class="article__title">
5.
gr.
</h3>

<p>
Gjöld
samkvæmt
3.
gr.
breytast
árlega
samkvæmt
vísitölu
byggingarkostnaðar,
sem
gildir
um
áramót.
</p>

<h3 class="article__title">
6.
gr.
</h3>

<p>
Gjalddagar
gatnagerðargjalds
skv.
3.
gr.
skulu
vera
sem
hér
segir:
</p>

<p>
Ef
um
er
að
ræða
leigulóð
í
eigu
sveitarfélagsins.
getur
sveitarstjórn
áskilið,
að
úthlutun
verði
háð
því,
að
full
skil
verði
gerð
innan
ákveðins
frests,
áður
en
byggingarleyfi
verður
veitt
eða
sérstakur
samningur
gerður
um
greiðslur.
</p>

<h3 class="article__title">
7.
gr.
</h3>

<p>
Gatnagerðargjald
skv.
4.
gr.
miðast
við
þá
vísitölu
byggingarkostnaðar,
sem
í
gildi
er,
þegar
lagning
bundins
slitlags
fer
fram.
Nú
breytist
vísitala,
meðan
verk
fer
fram,
og
skal
þá
miða
við
meðalvísitölu
á
því
tímabili,
sem
um
er
að
ræða.
</p>

<h3 class="article__title">
8.
gr.
</h3>

<p>
Nú
ákveður
sveitarstjórn
að
innheimta
gjald,
skv.
4.
gr.,
af
fasteignum
við
götu,
þar
sem
þegar
hefur
verið
lagt
slitlag
og
gangstétt,
skal
miða
gjaldið
við
þá
vísitölu,
sem
í
gildi
er,
þegar
samþykkt
þessi
öðlast
gildi.
</p>

<h3 class="article__title">
9.
gr.
</h3>

<p>
Gatnagerðargjald
samkvæmt
4.
gr.
greiðist
þannig
að
20%
greiðist,
þegar
lagningu
bundins
slitlags
er
lokið.
Eftirstöðvar
greiðast
með
jöfnum
árlegum
afborgunum
á
næstu
4
árum.
</p>

<p>
Gjalddagi
afborgana
er
1.
júli.
Vextir
10%
greiðast
eftir
á
sömu
gjalddögum.
</p>

<h3 class="article__title">
10.
gr.
</h3>

<p>
Sveitarstjórn
er
heimilt
að
hækka
eða
lækka
gjaldstuðla
samkv.
3.
gr.
og
4,
gr.
hvern
um
sig
eða
alla
í
einu
um
allt
að
50%
án
frekari
staðfestingar
ráðuneytis.
</p>

<h3 class="article__title">
11.
gr.
</h3>

<p>
Gatnagerðargjaldi
fylgir
lögveð
í
fasteign
þeirri
sem
það
er
lagt
á,
og
gengur
það
ásamt
vöxtum
fyrir
öllum
veðkröfum,
sem
á
eigninni
hvíla
og
tekur
einnig
til
vátryggingarfjár
eignarinnar.
</p>

<p>
Heimilt
er
að
taka
gjöld
samkvæmt
lögum
þessum
lögtaki
skv.
lögum
nr.
29/1885.
</p>

<p>
Samþykkt
þessi
er
samin
og
samþykkt
of
hreppsnefnd
Hvolshrepps
og
staðfestist
hér
með
samkvæmt
lögum
um
gatnagerðargjöld
nr.
51/1974
sbr.
lög
nr.
31/1975
til
að
öðlast
gildi
þegar
í
stað.
</p>

<p class="Dags">
Félagsmálaráðuneytið,
18.
nóvember
1975
</p>

<p class="FHUndirskr" align="center">
<strong>F.
h.
r.</strong>
</p>

<p align="center">
<strong>Jón
S.
Ólafsson.</strong>
</p>
